schema.js

```diff
@@ -1,23 +1,23 @@
-const summarySchema = new mongoose.Schema({
```

Comment:
Looks like this file is all commented out? Looks like the menu model is created in `restaruant.js` but missing the summary model.